assets/cms/components/Icon.vue

```diff
@@ -0,0 +1,36 @@
+<template functional>
+
+    <i class="icon" :class="[{
+            'fas': props.iconType === 'fontawesome-solid',
+            'far': props.iconType === 'fontawesome-regular',
+            'fab': props.iconType === 'fontawesome-brand',
+        }, props.icon]"
+        :style="{'color':props.color}"
+    />
+
+</template>
+
+<script>
+export default {
+    props: {
+        icon: {
+            type: String,
+            default: 'fa-exclamation-triangle'
```

Comment:
I don't think it makes sense to have a default value for this, instead just make it required